doc/classes/DampedSpringJoint2D.xml

```diff
@@ -78,12 +78,18 @@
 	</methods>
 	<members>
 		<member name="damping" type="float" setter="set_damping" getter="get_damping">
+			The spring joint's damping ratio.
+			When the two bodies move into different directions the system tries to allign them to the spring axis again.
+			This value defines how fast this happens, a higher value makes the allignment happen faster.
 		</member>
 		<member name="length" type="float" setter="set_length" getter="get_length">
+			The spring joint's maximum length.
```

Comment:
`The spring joint's maximum length. The two attached bodies cannot stretch it past this value. Default value: [code]add default[/code]`